client/a8c-for-agencies/sections/marketplace/hosting-overview-v3/style.scss

```diff
@@ -1,7 +1,7 @@
 $background-color: #05374A;
 
-.hosting-overview-v3.main.multi-sites-dashboard-layout {
-	.multi-sites-dashboard-layout__body {
+.hosting-overview-v3.main.hosting-dashboard-layout-layout {
```

Comment:
There are many unintended `hosting-dashboard-layout-layout` occurrences in this PR :)

Reply:
>maybe we can just call the directory hosting-dashboard? Since the parent directory is already layout

Good point, let's go with hosting-dashboard.